src/App.less

```diff
@@ -35,8 +35,9 @@ img {
   max-width: 800px;
 }
 
-.auth-form-col {
-  max-width: 500px;
+.auth-form-col, .org-form {
+    max-width: 500px;
+    margin: 0px auto;
```

Comment:
Can you please verify that this change will not affect to the elements that use the auth-form-col as style class of it?